lib/songkick/transport/curb.rb

```diff
@@ -63,6 +61,11 @@ def execute_request(req)
           connection.http(req.verb.upcase)
         end
 
+        if self.instrumenter
+          self.instrumentation_payload_extras[:connect_time] = connection.connect_time
```

Comment:
I think having them non-prefixed might be better. You could instrument Curb and HTTParty in the same way, without having to worry about the actual implementation in the instrumentation, only worrying about whether the attribute is there or not?